src/main/static/src/components/common/henkilo/labelvalues/Kansalaisuus.tsx

```diff
@@ -50,7 +53,7 @@ const Kansalaisuus = (props: Props) => {
                         label: koodi[props.locale],
                         optionsName: 'kansalaisuus',
                     })),
-                    selectValue: kansalaisuus.map((item) => item.kansalaisuusKoodi),
+                    selectValue: kansalaisuus.map((item) => item.kansalaisuusKoodi)[0],
```

Comment:
bugi korjattu vai luotu?

Reply:
eikö tän voisi kirjottaa kansalaisuus[0].kansalaisuuskoodi jos sen pitää olla näin?